draft-ietf-tls-tls13.md

```diff
@@ -2143,6 +2144,18 @@ be taken into account when designing new extensions:
 The client uses the "signature_algorithms" extension to indicate to the server
 which signature/hash algorithm pairs may be used in digital signatures.
 
+All clients MUST send a valid "signature_algorithms" extension containing
+at least one supported SignatureAndHashAlgorithm when offering any
+certificate authenticated cipher suites.
+Servers attempting to negotiate use of certificate authenticated cipher
+suites for TLS 1.3 or later MUST close the connection with a fatal
+"missing_extension" alert in the event that this extension is not present.
+(see {{mti-extensions}})
```

Comment:
Rephrase:
"Server MUST not negotiate use of a certificate authenticated cipher suite unless the client supplies a supported SignatureAndHashAlgorithm extension. If the client provides such a cipher suite without a SignatureAndHashAlgorithm extension the server SHOULD close..."